docs/specs/git.yml

```diff
@@ -30,13 +30,16 @@ inputs:
 outputs:
   docs/index.json: |
     {
-      "author_info": 
-        { "name": "charlie", "profile_url": "https://github.com/charlie", "display_name": "Charlie", "id": "3" },
-      "contributors": [
-        { "name": "bob", "profile_url": "https://github.com/bob", "display_name": "Bob", "id": "2" },
-        { "name": "alice", "profile_url": "https://github.com/alice", "display_name": "Alice", "id": "1" }
-      ],
-      "updated_at": "2018-01-01T00:00:00.0000000"
+      "_op_gitContributorInformation":
+      {
```

Comment:
also check `_op_gitContributorInformation.updated_at` and `_op_gitContributorInformation.updated_at_datetime` in at least one non-legacy test.